build/parse.y

```diff
@@ -566,11 +568,15 @@ primary_expr:
 	}
 |	'{' keyvalues '}'
 	{
+		exprValues := make([]Expr, len($2))
```

Comment:
prefer capacity make and append to avoid potential off by 1

exprValues := make([]Expr, 0, len($2))

then 
exprValues = append(exprValues, Expr(kv))

Reply:
Done.

Why can it be off by 1? Both slices are guaranteed to have the same length.

Reply:
I agree it won't here, but if you use [i] style you *might* make a mistake.  If you use append style, if you make a mistake, you just force a re-allocation